doc/src/modules/sets.rst

```diff
@@ -94,3 +94,24 @@ ComplexRegion
    :members:
 
 .. autofunction:: normalize_theta_set
+
+Iteration over sets
+^^^^^^^^^^^^^^^^^^^
+
+For set unions, `\{a, b\} \cup \{x, y\}` may have no problem to be
+comprehended like `\{a, b, x, y\}` regardless of the distinctiveness of
```

Comment:
Instead of "may have no problem..." perhaps use "can be treated as ... for iteration."